packages/core/src/forms/control/control.element.ts

```diff
@@ -170,6 +170,10 @@ export class CdsControl extends LitElement {
     return [baseStyles, styles];
   }
 
+  private get hasAriaTypeLabel() {
+    return this.inputControl?.hasAttribute('aria-label') || this.inputControl?.hasAttribute('aria-labelledby');
```

Comment:
Seems like line 174 would be a good candidate for a utility in `a11y.ts`. It would make it easier to unit test too. The private getter could still remain. Out of scope for this PR, IMO, but could you add a todo comment?

Reply:
Fixed, refactored it out to the a11y.ts file